static/js/tools/map/chart_loader.tsx

```diff
@@ -145,29 +157,47 @@ function loadChartData(
   populationData: { [dcid: string]: SourceSeries },
   isPerCapita: boolean,
   geoJsonData: GeoJsonData,
+  placeInfo: PlaceInfo,
   setChartData: (data: ChartData) => void
 ): void {
-  const dataValues = {};
+  const mapDataValues = {};
+  const breadcrumbDataValues = {};
   const sources: Set<string> = new Set();
   const statVarDates = {};
   for (const dcid in statVarData.stat) {
     if (_.isEmpty(statVarData.stat[dcid])) {
       continue;
     }
-    const statVarValue = statVarData.stat[dcid].value;
-    const importName = statVarData.stat[dcid].metadata.importName;
-    sources.add(statVarData.metadata[importName].provenanceUrl);
-    statVarDates[dcid] = statVarData.stat[dcid].date;
+    let statVarValue = statVarData.stat[dcid].value;
     if (isPerCapita && dcid in populationData) {
```

Comment:
this could be more straightforward to read if it was structured:

```
if (isPerCapita) {
  if (dcid in population) {
  ...
  } else {
  ...
  }
}
```